packages/babel-types/src/definitions/core.js

```diff
@@ -279,41 +302,44 @@ defineType("FunctionExpression", {
   inherits: "FunctionDeclaration",
   aliases: ["Scopable", "Function", "BlockParent", "FunctionParent", "Expression", "Pureish"],
   fields: {
```

Comment:
It doesn't look like "inherits" merges things? `opts.fields = opts.fields || inherits.fields || {};`